app/src/main/java/de/westnordost/streetcomplete/quests/religion/AddReligionToWaysideShrine.java

```diff
@@ -0,0 +1,28 @@
+package de.westnordost.streetcomplete.quests.religion;
+
+import android.support.annotation.NonNull;
+
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+
+public class AddReligionToWaysideShrine extends AbstractAddReligionToQuestType
+{
+    @Inject public AddReligionToWaysideShrine(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+    @Override protected String getTagFilters()
+    {
+        return "nodes, ways, relations with historic=wayside_shrine and" +
```

Comment:
BTW why do you always list nodes, ways and relations explicitly. (also are there even ways with that?) Is not it also possible to omit that thing, I mean there are no more types than these three in OSM AFAIK, is not it? (In the Overpass-turbo query wizard, I can omit this.)